app/src/free/java/org/mozilla/fenix/settings/about/OssLicensesMenuImpl.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.about
+
+import android.content.Context
+import android.content.Intent
+
+class OssLicensesMenuImpl : OssLicensesMenu {
+    override fun getIntent(context: Context?, title: String): Intent {
+        // This menu shouldn't displayed on the free variant anyways.
```

Comment:
Why not? AFAIK open source libraries should be fine to leave in, and we'll need to credit them to use them in the app.

Reply:
I'm still a bit confused here. Why can't we use the Google library? As far as I can tell its open source and just figures out the play services libraries we use, rather than depending on them. 

If we need to change the license implementation I think it would be nice to have it split into a separate PR so its easier to review.

Reply:
Actually it **is** confusing. The gradle plugin that lists the dependencies is open-source and under Apache 2 license, but the library that is shipped with the APK is proprietary and, as far as I can tell, closed-source. Morever, the library has dependencies to gms (which you can easily see with dexdump) which is unsuitable for OSS anyways.

Agreed that it would make sense to split this into a separate PR though.

Reply:
Got it, thanks for breaking it down. Using the most popular FLOSS version instead makes sense to me.

Reply:
#13767